src/Strategy/CommonClassesStrategy.php

```diff
@@ -120,15 +136,9 @@ public static function getCandidates($type)
                     trigger_error(sprintf('Got exception "%s (%s)" while checking if a PSR-18 Client is available', get_class($e), $e->getMessage()), E_USER_WARNING);
                 }
             }
-
-            return $candidates;
-        }
-
-        if (isset(self::$classes[$type])) {
-            return self::$classes[$type];
```

Comment:
don't we still need this bit?

Reply:
No, it was replaced by the the foreach at the top of the method.